stdlib/Random/src/RNGs.jl

```diff
@@ -321,7 +321,7 @@ function make_seed()
         println(stderr,
                 "Entropy pool not available to seed RNG; using ad-hoc entropy sources.")
         seed = reinterpret(UInt64, time())
-        seed = hash(seed, UInt64(getpid()))
+        seed = hash(seed, UInt(getpid()))
```

Comment:
Actually, while you are at it, there is another `hash` call below... the result of `parse` is `UInt64`, I suggest to just take it modulo `UInt`, i.e. `hash(seed, parse(UInt64, ...) % UInt)`, to be on the safe side. This path is guarded by a `try-catch` block, so it's less crucial, but let's clean it up too!

Reply:
ops, thanks for pointing this out !
I decided to use modulo in both places for (local) consistency.